scripts/trigger-pipeline.sh

```diff
@@ -45,6 +45,6 @@ PARAM_USER=$(printf '%s\n' "${!PARAM_USER_ENV_VAR}")
 PARAM_REPO=$(printf '%s\n' "${!PARAM_REPO_ENV_VAR}")
 PARAM_BRANCH=$(printf '%s\n' "${!PARAM_BRANCH_ENV_VAR}")
 
-if [[ "$SKIP_TRIGGER" == "false" ]]; then
+if [[ "$SKIP_TRIGGER" == "0" ]]; then
```

Comment:
i think some people use the script directly so let us check both 0 and false
```suggestion
if [[ "$SKIP_TRIGGER" == "0" || "$SKIP_TRIGGER" == "false" ]]; then
```